app/helpers/notifications_helper.rb

```diff
@@ -2,11 +2,34 @@ module NotificationsHelper
   def notification_image(notification)
     case notification.type
     when 'new_discussion_post'
+      # Face of the discussion post user
```

Comment:
What does `Face of` mean?

Reply:
Oh. Sorry. It's obvious now.

Reply:
This method was so hard to make sense of without the comments. I'd really like to refactor the way we do this at some point, but it's tricky.